widget/entry.go

```diff
@@ -106,7 +107,7 @@ func (e *entryRenderer) Objects() []fyne.CanvasObject {
 // Entry widget allows simple text to be input when focused.
 type Entry struct {
 	baseWidget
-
+	fyne.ShortcutHandler
```

Comment:
I think this exposes the HandleShortcut and AddShortcut as part of the public Entry API - which it shouldn't do...
Perhaps make it an unexported member instead?